bundles/org.openhab.binding.kaleidescape/README.md

```diff
@@ -430,4 +432,26 @@ then
     }
     kactions.sendKCommand("GET_CONTENT_DETAILS:" + z1_Music_AlbumHandle.state.toString + ":")
 end
+
+rule "Movie Search"
+when
+    Item z1_MovieSearch received update
+then
+    if (z1_MovieSearch.state != NULL && z1_MovieSearch.state.toString.length > 0) {
```

Comment:
If `received update`, use `newState` instead of `z1_MovieSearch`.

Reply:
ok, fixed on both readmes